package/MDAnalysis/analysis/msd.py

```diff
@@ -281,6 +282,10 @@ class EinsteinMSD(AnalysisBase):
         the MSD. Otherwise, use the simple "windowed" algorithm.
         The tidynamics package is required for `fft=True`.
         Defaults to ``True``.
+    non_linear : bool
+        If ``True``, calculates MSD for trajectory where frames are
+        non-linearly dumped. To use this set `fft=False`.
+        Defaults to ``False``.
```

Comment:
Indicate when this new kwarg was added so that users understand which version of the code they need in order to use it
```suggestion
        Defaults to ``False``.
       
        .. versionadded:: 2.10.0
       
       
```
(needs 2 newlines below so that formatting works, manually check the rendered docs from the status checks)